static/templates/gear_menu.hbs

```diff
@@ -1,185 +1,196 @@
 <ul class="nav" role="navigation">
-    <li class="dropdown actual-dropdown-menu" id="gear-menu">
+    <div class="dropdown actual-dropdown-menu" id="gear-menu">
         <a id="settings-dropdown" tabindex="0" role="button" class="dropdown-toggle" data-target="nada" data-toggle="dropdown" title="{{t 'Menu' }} (g)">
             <i class="fa fa-cog settings-dropdown-cog" aria-hidden="true"></i>
         </a>
-        <ul class="dropdown-menu" role="menu" aria-labelledby="settings-dropdown">
-            <li class="org-info org-name">{{realm_name}}</li>
-            <li class="org-info org-url">{{realm_uri}}</li>
-            {{#if is_self_hosted }}
-                <li class="org-info org-version">
-                    <a href="#about-zulip" role="menuitem">{{version_display_string}}</a>
-                </li>
-                {{#if server_needs_upgrade }}
-                <li class="org-info org-upgrade small-font-size">
-                    <a href="https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html" target="_blank" rel="noopener noreferrer" role="menuitem">{{t 'Upgrade to the latest release' }}</a>
-                </li>
+        <nav class="dropdown-menu" role="menu" aria-labelledby="settings-dropdown">
```

Comment:
These are a lot of changes!

What's the story with the refactoring in this file? One comment is that we want to remove the `nav` CSS class -- that's a Bootstrap thing and we're trying to remove all use of Bootstrap. I think it might be most helpful if you were to do a preparatory PR that changed details like the `li` -> `div` at the top with separate commits, with no functional/visual changes, and then have individual commits for things like "Reorder the settings" and then "Redo the visual styling".

This will be important to turning this into a [reviewable pull request](https://zulip.readthedocs.io/en/latest/contributing/reviewable-prs.html) -- since it'll be possible to see that, for example, you've preserved all the HTML conditionals as originally written in each commit.

Check out the [Zulip commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for more details.

Reply:
Split commit.
As for the bootstrap .nav, I would like to keep it in this PR.
This is because it affects the design of the buttons in the header and so on.

The PR for adding the personal menu dropdown will also make changes to the header.
So I would like to work on this in there.